web/app/webpack.config.js

```diff
@@ -64,10 +66,17 @@ module.exports = smp.wrap({
   },
   plugins: [
     // new BundleAnalyzerPlugin(), // uncomment to analyze bundle size
+    new CleanWebpackPlugin(['dist']),
+    new HtmlWebpackPlugin({
+      inject: false,
+      filename: "index_bundle.js.out",
+      template: 'index_bundle.js.lodash.tmpl',
```

Comment:
Sorry, one more clarifying question: why does this file have "lodash" in the name? Is there anything lodash-specific about the file or the `HtmlWebpackPlugin` that's using it? Would it make sense to rename it to `index_bundle.js.tmpl`?

Reply:
Hmm, `index_bundle.js.lodash.tmpl` is a "lodash template", which is what `HtmlWebpackPlugin` defaults to:
https://github.com/jantimon/html-webpack-plugin/blob/master/docs/template-option.md#1-dont-set-any-loader

That said, according to the lodash docs, this should be equivalent to ES2015 or ERB:
https://lodash.com/docs/4.17.11#templateSettings

I don't feel super strongly about this, was erring on the side explicitness. During development I was trying several non-default `HtmlWebpackPlugin` templating systems, and it was helpful to know what type of templating each file was written for.

Reply:
Ahh, ok, thanks for the additional explanation. Definitely no need to change it. I didn't realize that lodash supported it's own templating and that's what we're using. TIL.